### PR TITLE
Men 15

### PR DESCRIPTION
**Pull request by aleumyu**
finished profile & favorite components: users can see their profile with favorite list. favorite list can be removed.

Thing to be done in the future
- More styling
- if the profile view is not from current logged in user, then the favorite list should not be shown

### CODE REVIEW

File: api/routes/index.js [hunk 1]
```diff
@@ -7,12 +7,40 @@ router.get('/', function(req, res, next) {
   res.send({ response: 'Express reponse' });
 });
 
+router.get('/api/v1/users/:id', (req, res, next) => {
+  db(`SELECT * FROM user WHERE userId=${req.params.id};`)
```
Comment (dankostiuk) [inline api/routes/index.js:11]: What happens if `req.params.id` is null or undefined?

File: api/routes/index.js [hunk 2]
```diff
@@ -7,12 +7,40 @@ router.get('/', function(req, res, next) {
...
+    .then(results => {
+      if (results.error) {
+        res.status(500).send(results.error);
+      }
+      console.log('results: ' + JSON.stringify(results.data));
+      res.send(results.data);
+    })
+});
+
+//`SELECT u.userId, photo, industry, jobType, years, intro, location, role, meeting, firstName, lastName, i.interestTag FROM user u INNER JOIN interests i ON u.userId=i.userId WHERE u.userId=${req.params.id};
```
Comment (dankostiuk) [inline api/routes/index.js:32]: Make sure to remove commented code :)

File: client/src/components/profile.js [hunk 3]
```diff
@@ -1,27 +1,114 @@
...
 
-  callAPI() {
-    // calls our backend on port 9000, modify this!
-    fetch("http://localhost:9000/")
-      .then(res => res.json())
-      .then(res => console.log(res));
+  removeFavoite(event, i) {
+    const {id} = this.props.match.params;
+    console.log(this.state.favoritesList[i]);
+    event.preventDefault();
```
Comment (dankostiuk) [inline client/src/components/profile.js]: Maybe add this to the top of the method

File: client/src/components/profile.js [hunk 4]
```diff
@@ -1,27 +1,114 @@
...
+                          
+                          {
+                          (e.role === 0) ? <p>{'mentor'}</p>
+                          : (e.role === 1) ? <p>{'mentee'}</p>  
+                          : (e.role === 2) ? <p>{'mentor' + 'mentee'}</p>
+                          : <p></p>
+                          }
+
+                          <p>{e.years} year</p>
+                          <p>{e.indusrty}</p>
```
Comment (dankostiuk) [inline client/src/components/profile.js]: Typo in `industry` :)

**Review by dankostiuk — COMMENTED** [review]
See my comments, otherwise looks good!

File: client/src/components/profile.js (hunk 3 above)
Comment (aleumyu) [reply client/src/components/profile.js]: done!

File: client/src/components/profile.js (hunk 4 above)
Comment (aleumyu) [reply client/src/components/profile.js]: done!

**Pull request merged**